hack/collector-default

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system_info : " `uname -a` >> hostinfo
+
+    # Boot Configuration
+    cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
```

Comment:
`kernel_config` instead.

Reply:
I have renamed this file to `kernel_config`.Thanks.
![image](https://user-images.githubusercontent.com/17548901/211154957-2bd845c1-584e-4d4d-97cf-ed2f24c8aeb3.png)